### ISSUE TITLE
More DS5002FP issues in gaelco2.c

### ISSUE THREAD

**Issue by mc68k** [mamedev/mame#6098]
With the latest release, the protected sets of Alligator Hunt are showing some odd behavior:

1.  The ships flying in during the intro are delayed, giving it an odd 'missing sprites' feel.
2.  When the lizard appears, the left side appears first, the second side a second later, followed by the head a second after that.
3.  At the title screen, the scrolling effect on Alligator takes several seconds to appear.  The game goes to demo mode before 'hunt' can even be shown.
4.  During the demo, player is shown shooting at empty air, with sprites appearing about 3 seconds after the demo has started.

In short, it seems like the sprites in the intro are running half the normal rate, while other graphics are running normally.

Back checking, .215 the protected sets are normal and match the unprotected sets.  In .216 and .217, the lagging appeared.

Adding config.set_perfect_quantum("gaelco_ds5002fp:mcu"); right after line 1221 in gaelco2.c corrects the protected set's behavior.  That is, assuming this is a timing issue similar to slow gameplay in TH Strikes Back and not just covering up something else.  See: https://github.com/mamedev/mame/commit/a52d527a1cc6dcf5bc48268d0b0bd451aaeaf814

**Comment by DavidHaywood** [comment]
yes, all of these should be running with high / perfect interleave.  I don't know who changed them, but it was always going to break things.

**Comment by mc68k** [comment]
I tested other games in the driver with the ds5002fp but couldn't immediately see anything off, so I didn't mess with those.  But not all of those have an unprotected set to check against e.g. target hits.

**Comment by DavidHaywood** [comment]
visible effects are irrelevant, they need tight communication otherwise things will end up delayed by multiple frames even if you don't notice them.

**Comment by mc68k** [comment]
I've added the quantum lines to the drivers that used the 5002.  I opened a pull request with the changes, but seemed to have put it in the wrong directory even after reading the github help several times.  I tried though.

**Comment by DavidHaywood** [comment]
should be possible to close this now
